routes/articles.js

```diff
@@ -2,13 +2,34 @@ const router = require('express').Router();
 let Article = require('../models/Article.model');
 
 router.route('/').get((req, res) => {
+    console.log("bruh / check");
     Article.find()
         .then(articles => res.json(articles))
         .catch(err => res.status(400).json('Error: ' + err));
-});
+})
+
+router.route('/:_id').get((req, res) => {
+    const id = req.params;
+
+    Article.findById(id)
+        .then(article => res.json(article))
+        .catch(err => res.status(400).json('Error: ' + err));
+})
+
+router.route('/:_id').delete((req, res) => {
+    console.log("bruh /:_id delete check");
+
+    const id = req.params;
+
+    Article.findByIdAndDelete(id)
+        .then(() => res.json("Article Deleted"))
+        .catch(err => res.status(400).json('Error: ' + err));
+})
 
 router.route('/add').post((req, res) => {
-   // console.log("Request: " + JSON.stringify(req.body));
+    console.log("bruh add check");
```

Comment:
Same thing here, comment out and give a better name representing what you're checking